pkg/tcl/testworkflowstcl/testworkflowprocessor/constants.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2024 Testkube.
+//
+// Licensed as a Testkube Pro file under the Testkube Community
+// License (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//	https://github.com/kubeshop/testkube/blob/main/licenses/TCL.txt
+
+package testworkflowprocessor
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	corev1 "k8s.io/api/core/v1"
+
+	testworkflowsv1 "github.com/kubeshop/testkube-operator/api/testworkflows/v1"
+	"github.com/kubeshop/testkube/cmd/kubectl-testkube/commands/common"
+)
+
+const (
+	defaultImage        = "busybox:1.36.1"
```

Comment:
Default is already configurable on the TestWorkflow level, as you may set top-level `container: { image: "another-default-image" }`. We could add some environment variable on API Server level to default the default though, if you want 🙂 I'm not sure if it's worth too, considering that we will likely have some `global` TestWorkflowTemplate, that will be automatically added to all of the TestWorkflows, where it could be configured (with top-level `container` settings, as above).

When it comes to the other properties, like internal or data directory, I'm considering that we could add in the future option to pass another property to the TestWorkflow, like `internal: { dataDirectory: {path string, volumeName string} }` or something like that

Reply:
ok, let's see, if we have any questions about it